frontendservice/src/main.js

```diff
@@ -16,6 +16,7 @@ import firebase from "firebase";
 import App from "./App.vue";
 import "./registerServiceWorker";
 import router from "./router";
+import "bulma/css/bulma.css";
```

Comment:
Again, this is not required I think. We have been able to use Bulma features without this statement.